### ISSUE TITLE
Getting error while executing `npm install && npm start`

### ISSUE THREAD

**Issue by bhaskar253** [bloodorca/hollow#2]
**NodeJS version: 14.8.0**

> hollowclient@1.0.0 start /home/test/Desktop/rough/my-demo-proj
> webpack-dev-server --open

ℹ ｢wds｣: Project is running at http://localhost:8080/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from ./docs
✖ ｢wdm｣: Hash: 7cf6629f00ab9b881562
Version: webpack 4.44.1
Time: 1226ms
Built at: 09/08/2020 5:59:02 AM
     Asset       Size  Chunks             Chunk Names
 bundle.js    880 KiB    main  [emitted]  main
index.html  710 bytes          [emitted]  
Entrypoint main = bundle.js
[0] multi (webpack)-dev-server/client?http://localhost:8080 ./src/index.js 40 bytes {main} [built]
[./node_modules/ansi-html/index.js] 4.16 KiB {main} [built]
[./node_modules/ansi-regex/index.js] 135 bytes {main} [built]
[./node_modules/html-entities/lib/index.js] 449 bytes {main} [built]
[./node_modules/loglevel/lib/loglevel.js] 8.65 KiB {main} [built]
[./node_modules/strip-ansi/index.js] 161 bytes {main} [built]
[./node_modules/webpack-dev-server/client/index.js?http://localhost:8080] (webpack)-dev-server/client?http://localhost:8080 4.29 KiB {main} [built]
[./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.51 KiB {main} [built]
[./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.53 KiB {main} [built]
[./node_modules/webpack-dev-server/client/utils/createSocketUrl.js] (webpack)-dev-server/client/utils/createSocketUrl.js 2.91 KiB {main} [built]
[./node_modules/webpack-dev-server/client/utils/log.js] (webpack)-dev-server/client/utils/log.js 964 bytes {main} [built]
[./node_modules/webpack-dev-server/client/utils/reloadApp.js] (webpack)-dev-server/client/utils/reloadApp.js 1.59 KiB {main} [built]
[./node_modules/webpack-dev-server/client/utils/sendMessage.js] (webpack)-dev-server/client/utils/sendMessage.js 402 bytes {main} [built]
[./node_modules/webpack/hot sync ^\.\/log$] (webpack)/hot sync nonrecursive ^\.\/log$ 170 bytes {main} [built]
[./src/index.js] 950 bytes {main} [built] [failed] [1 error]
    + 18 hidden modules

ERROR in ./src/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: Unexpected token (19:10)

  17 |         windowDrag.onDragLeave = () => this.setState({ dragging: false })
  18 |     }
> 19 |     state = {
     |           ^
  20 |         gameFile: "", 
  21 |         gameFileOriginal: "",
  22 |         editing: false,

Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html] 871 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 530 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.

**Comment by bloodorca** [comment]
Apologies for the wait, I didn't get a notification for this. I don't remember much of this project. I think to build it...

1. cd into the project root folder. 
2. npm install         # install dependencies 
3. npm run build    # build docs folder 
4. open docs/index.html in your browser.

**Comment by bhaskar253** [comment]
That's what I too tried but as you have not added **"package-lock.json"**. I was getting inconsistent versions of some of the dependencies related to babel and hence the above errors.

After much of hassle finally, I was able to fix the issue by just creating the new npm project with all the latest dependencies and then copying and modifying some of your code which was not working for me due to some change in React event handlers. 

Anyways thanks a lot for writing this wonderful code. I was able to learn a lot of things from it.
